package.json

```diff
@@ -98,7 +98,7 @@
     "js-yaml": "^4.1.0",
     "mocha": "^10.2.0",
     "mocha-sinon": "^2.1.2",
-    "mongodb-client-encryption": "^6.0.0-alpha.1",
+    "mongodb-client-encryption": "6.0.0-alpha.1",
```

Comment:
Can we bump this to alpha 2 now that it's out?

Reply:
alpha.2 introduced the bug that's making the CI red, rolling back fixes it locally but all of EVG disagrees 😅 I'm not sure how the code is getting a hold of the alpha.2 version, I thought we only installed from devDeps now.

Reply:
Context: https://jira.mongodb.org/browse/NODE-5516

Now that we track the package in our devDeps we can get rid of the peppering of manual installations. cc @baileympearson